app/komponenter/tekstBlokk/tekstBlokk.tsx

```diff
@@ -0,0 +1,44 @@
+import { TypografiTyper } from '~/typer/typografi';
```

Comment:
Ser at det er noen warnings her som sikkert er greit å fikse. Jeg er ikke helt fan av warnings i eslint fordi jeg tenker at enten følger man konvensjon eller ikke. 

Tenker det kan være greit å endre slik at max warnings er satt til 0. Har mulighet til å gjøre det? Det må endres både i commit-hooken og i actionen.
![image](https://github.com/bekk/nav-familie-endringsmelding/assets/17828446/bcc4735e-3c9e-489b-8367-996d13546b4b)

Reply:
Vi deaktiverte regelen for type checking ved import og la til at max warning er 0. Se [0d91d61](https://github.com/bekk/nav-familie-endringsmelding/pull/13/commits/0d91d6175262b4086bf085015979ef793d5c04a8)